### PR TITLE
#2535 Updated new OSS stacks

### PR DESCRIPTION
**Pull request by neetapimpalnerkar**
Please review #2535.

### CODE REVIEW

File: source_en/Overview/FrameworkStack.rst [hunk 1]
```diff
@@ -530,13 +524,13 @@ List of OSS being used in version 5.2.0.RELEASE.
...
       - \*
       - \*3
     * - Tiles
       - org.apache.tiles
       - tiles-core
-      - 3.0.5
+      - 3.0.7
       - \*
       - \*3
     * - Tiles
```
Comment (btundes) [inline source_en/Overview/FrameworkStack.rst]: @neetapimpalnerkar 
In Japanese file the versions of following libaries are upgraded to 3.0.7. However, in english translated file its the same old 3.0.5. Please confirm and modify in the translated file .
1. tiles-jsp
2. tiles-servlet
3. tiles-template

Comment (neetapimpalnerkar) [inline source_en/Overview/FrameworkStack.rst]: Ok I will check it

**Comment by neetapimpalnerkar** [comment]
@btundes Sandeep san...I have made the changes. Please review.

**Comment by btundes** [comment]
@neetapimpalnerkar 
Thanks for the update Neeta-san. The same changes also needs to be done for the 
2. tiles-servlet
3. tiles-template  libraries as mentioned in the previous comment.

**Comment by neetapimpalnerkar** [comment]
@btundes Done.

**Comment by btundes** [comment]
@neetapimpalnerkar 
:+1: Thanks for the modification Neeta-san.  All seems to be good now.

**Pull request merged**